Rakefile

```diff
@@ -11,12 +12,15 @@ task default: "spec"
 require 'jars/installer'
 desc 'Install the JAR dependencies to vendor/'
 task :install_jars do
-  # If we don't have these env variables set, jar-dependencies will
-  # download the jars and place it in $PWD/lib/. We actually want them in
-  # $PWD/vendor
-  ENV['JARS_HOME'] = Dir.pwd + "/vendor/jar-dependencies/runtime-jars"
-  ENV['JARS_VENDOR'] = "false"
-  Jars::Installer.new.vendor_jars!(false)
+  # We actually want jar-dependencies will download the jars and place it in
+  # vendor/jar-dependencies/runtime-jars
+  if Gem::Version.new(Jars::VERSION) >= Gem::Version.new('0.3.4')
```

Comment:
Because I want to make it compatible with the version 0.3.2, I add this this conditional here.
I already removed this conditional.